Xls/Reader/excel_openpyxl.py

```diff
@@ -25,7 +25,11 @@ def run(argv):
     for row in sheet.iter_rows(row_offset=int(args.start) - 1):
         current_row_read = []
         for cell in row:
-            current_row_read.append(cell.value.encode('utf-8'))
+            value = cell.value
+            if isinstance(value, (int, long)):
```

Comment:
of course, it`s up to you, but from my POV, it is better to make custom config once than adding all these annoying checks and fixes )